aclk/aclk_lws_wss_client.c

```diff
@@ -315,7 +319,7 @@ static int aclk_lws_wss_callback(struct lws *wsi, enum lws_callback_reasons reas
             break;
 
         default:
-            error("Unexecpted callback from libwebsockets %s", aclk_lws_callback_name(reason));
+            error("Unexpected callback from libwebsockets %s", aclk_lws_callback_name(reason));
```

Comment:
I am not sure we should log every callback even as LWS library grows callbacks can be added that we are simply not interested in. But up to you

Reply:
That is a fair point. My reasoning is as follows: we need to be sure about every callback that is generated by an external dependency. In a sense this part of the code is there to enforce white-listing of those events, i.e. one of us has sat and read the docs and made an explicit effort to put it onto the "don't care" list. If the library changes in the future and new callbacks are generated then somebody needs to perform that explicit step and this logging will force that to happen.